test/hash-links.md

```diff
@@ -14,12 +14,42 @@ The title is [Foo](#foo).
 
 The second section is [Bar](#bar).
 
+To test a failure. Link that [does not exist](#does-not-exist).
+
 ## Uh, oh
 
 There is no section named [Potato](#potato).
 
 There is an anchor named [Tomato](#tomato).
 
-## Header with special char ✨
+## Header with special char at end ✨
```

Comment:
Also try these - as they actively error for me in 3.12

```md
## --docker
[--docker](#--docker)

## Step 7 - Lint & Test
[Step 7 - Lint \& Test](#step-7---lint--test)

## Product Owner / Design Approval
[Product Owner / Design Approval](#product-owner--design-approval)
```

These are as they are created by markdown all in one extension in vscode

Reply:
I have this added to the tests. And it works.

Reply:
That's great news, so this fix isn't in 3.12 - and just needs merging and releasing?

Do we know when it will be released?

Reply:
@tcort any ideas for a timeline?